src/peergos/server/tests/MultiUserTests.java

```diff
@@ -488,10 +490,43 @@ public void followPeergos() throws Exception {
         UserContext u1 = PeergosNetworkUtils.ensureSignedUp("peergos", "testpassword", network, crypto);
         UserContext u2 = PeergosNetworkUtils.ensureSignedUp("w", "w", network, crypto);
 
+        u2.sendFollowRequest(u1.username, SymmetricKey.random()).get();
```

Comment:
this shouldn't be here. Users automatically friend "peergos" during signup

Reply:
Got it. Will remove.